### PR TITLE
Fix typo in "kubectl create role" documentation translations

### PR DESCRIPTION
**Pull request by seh**
### What this PR does / why we need it

Though the original source of this documentation in file [_pkg/kubelet/cmd/create_role.go_](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/create_role.go) has since been corrected in commit 877bf54465947c32ed46dd142866ca9e4da1ab9a, the misspelling persisted in these translation files. Note that the translations no longer the primary source, and are in need of a broader update.

I noted the original problem at its point of origin [here](https://github.com/kubernetes/kubernetes/commit/0a2876d39e37cef982d9b61c32131a2de180cc7f#commitcomment-22383356).

### Special notes for your reviewer

While correcting these translations certainly doesn't _hurt_, the real problem is that the translations are lagging far behind the primary documentation still written in the Go source files.

### CODE REVIEW

**Comment by liggitt** [comment]
does `hack/update-translations.sh` modify these or produce other artifacts that need to be committed?

**Comment by seh** [comment]
It looks like [_hack/update-translations.sh_ might write these files](https://github.com/kubernetes/kubernetes/blob/master/hack/update-translations.sh#L59-L82). I have never run it. Are you suggesting that I run that script and see if it corrects this problem?

File: translations/kubectl/default/LC_MESSAGES/k8s.po [hunk 1]
```diff
@@ -229,7 +229,7 @@ msgid ""
 "resource=pods\n"
 "\n"
 "\t\t# Create a Role named \"pod-reader\" with ResourceName specified\n"
-"\t\tkubectl create role pod-reader --verb=get --verg=list --verb=watch --"
+"\t\tkubectl create role pod-reader --verb=get --verb=list --verb=watch --"
```
Comment (liggitt) [inline translations/kubectl/default/LC_MESSAGES/k8s.po]: also on line 242

File: translations/kubectl/en_US/LC_MESSAGES/k8s.po [hunk 2]
```diff
@@ -229,7 +229,7 @@ msgid ""
 "resource=pods\n"
 "\n"
 "\t\t# Create a Role named \"pod-reader\" with ResourceName specified\n"
-"\t\tkubectl create role pod-reader --verb=get --verg=list --verb=watch --"
+"\t\tkubectl create role pod-reader --verb=get --verb=list --verb=watch --"
```
Comment (liggitt) [inline translations/kubectl/en_US/LC_MESSAGES/k8s.po]: also on line 242

**Pull request closed without merging**